packages/block-library/src/post-title/block.json

```diff
@@ -5,6 +5,15 @@
 		"postId",
 		"postType"
 	],
+	"attributes": {
+		"align": {
+			"type": "string"
+		},
```

Comment:
Do you think instead of handling the alignment manually, we could use the "align" support flag? Other blocks do that like "button".

Reply:
The support flag adds block level alignment using floats though doesn't it? Other heading blocks use text alignment, so it seems more appropriate to replicate that for the post title block.

Reply:
Yeah, we might want to add a support flag for text alignment later on.

Reply:
Ohh, I misunderstood this, I thought it was adding the text alignment support. Makes me wonder if we should rename the attribute or not? Do we already have blocks that support both text and block alignments? How do we name these things there?

Reply:
Yes, it's super confusing. It should be `textAlign`.